src/app/tests/suites/certification/PICS.yaml

```diff
@@ -2735,18 +2735,64 @@ PICS:
     - label: "Does the DUT(server) support the ExpiringUserTimeOut attribute?"
       id: DRLK.S.A0035
 
+    - label:
+          "Does the DUT(server) support the AliroReaderVerificationKey attribute
+          and is ReadOnly"
+      id: DRLK.S.A0080
+
+    - label:
+          "Does the DUT(server) support the AliroReaderGroupIdentifier attribute
+          and is ReadOnly"
+      id: DRLK.S.A0081
+
+    - label:
+          "Does the DUT(server) support the AliroReaderGroupSubIdentifier
+          attribute and is ReadOnly"
+      id: DRLK.S.A0082
+
+    - label:
+          "Does the DUT(server) support the
+          AliroExpeditedTransactionSupportedProtocolVersions attribute and is
+          ReadOnly"
+      id: DRLK.S.A0083
+
+    - label:
+          "Does the DUT(server) support the AliroGroupResolvingKey attribute and
+          is ReadOnly"
+      id: DRLK.S.A0084
+
+    - label:
+          "Does the DUT(server) support the AliroSupportedBLEUWBProtocolVersions
+          attribute and is ReadOnly"
+      id: DRLK.S.A0085
+
+    - label:
+          "Does the DUT(server) support the AliroBLEAdvertisingVersion attribute
+          and is ReadOnly"
+      id: DRLK.S.A0086
+
+    - label:
+          "Does the DUT(server) support the
+          NumberOfAliroCredentialIssuerKeysSupported attribute and is ReadOnly"
+      id: DRLK.S.A0087
+
+    - label:
+          "Does the DUT(server) support the NumberOfAliroEndpointKeysSupported
+          attribute and is ReadOnly"
+      id: DRLK.S.A0088
+
     #
     #Server ReadOnly attribute
     #
     - label:
           "Does the DUT(server) support the EnableLocalProgramming attribute and
           is ReadOnly"
-      id: DRLK.S.A0028.ReadOnly
+      id: DRLK.S.A0028
```

Comment:
There's already a PICS with id DRLK.S.A0028 defined above.  Why is this here?

Reply:
This came as legacy one. we should not have touched that i.e. DRLK.S.A0028.ReadOnly to DRLK.S.A0028 will update the PR.